config/application.yml.default

```diff
@@ -20,6 +20,7 @@ aamva_cert_enabled: true
 aamva_supported_jurisdictions: '["AL","AR","AZ","CO","CT","DC","DE","FL","GA","HI","IA","ID","IL","IN","KS","KY","MA","MD","ME","MI","MO","MS","MT","NC","ND","NE","NJ","NM","NV","OH","OR","PA","RI","SC","SD","TN","TX","VA","VT","WA","WI","WV","WY"]'
 aamva_verification_request_timeout: 5.0
 aamva_verification_url: https://example.org:12345/verification/url
+account_creation_device_profiling_enabled: false
```

Comment:
Over in IdV, `proofing_device_profiling` is an enum that accepts `:enabled`, `:disabled`, and `:collect_only`, the idea being that sometimes we want to run device profiling, but not necessarily have it affect the results of an identity verification (that is, we run it and log the results so we can see what it would have done). Y'all *might* consider something similar here, but it's going to depend on your use case.

Reply:
oo good idea, we might do it just to ensure we have similar formatting for simplicity.